docs/index.rst

```diff
@@ -1,21 +1,7 @@
-.. include:: README.rst
-
-.. include:: multiprocessing.rst
-
-Api Reference
+API Reference
```

Comment:
Please add this file to the ignores list

Reply:
Will do. I plan to keep these deletions though as there is no version 2.x release.

I'm also looking into removing duplicate code in the repo: https://github.com/googleapis/python-monitoring-dashboards/tree/master/google/monitoring

Reply:
Initially I thought there was no version 2.x release. My mistake.

Reply:
Done.